src/transport_picoquic.cpp

```diff
@@ -810,12 +799,24 @@ PicoQuicTransport::enqueue(const TransportContextId& context_id,
             metrics.enqueued_objs++;
             stream_cnx->second.tx_data->push(bytes, ttl_ms, priority);
 
+            if (!stream_id) {
+                picoquic_runner_queue.push([=, this]() {
```

Comment:
I see this for everything pushed to the runner queue: Do we want to capture by value? That does mean a copy of everything used (except for `this`) every time we push to the queue.

Reply:
Good catch. It was a cut/paste in. I'll update that.